vscode-dotnet-runtime-library/src/Acquisition/DotnetConditionValidator.ts

```diff
@@ -29,22 +29,28 @@ export class DotnetConditionValidator implements IDotnetConditionValidator
     {
         const hostArch = await this.getHostArchitecture(dotnetExecutablePath, requirement);
 
-        if (requirement.acquireContext.mode === 'sdk') {
+        if (requirement.acquireContext.mode === 'sdk')
+        {
             const availableSDKs = await this.getSDKs(dotnetExecutablePath);
-            if (availableSDKs.some((sdk) => {
+            if (availableSDKs.some((sdk) =>
+            {

```

Comment:
Nit: Same line, and maybe replace `(a) => { return statement }` with `(a) => statement`?